src/backplanes/backplanes.py

```
@@ -38,142 +37,63 @@ def generate_backplanes_image_files(
+        # TODO We only support snapshots for backplane generation for now
+        obs = obs_class.from_file(image_path, extfov_margin_vu=(0, 0))
+        if not isinstance(obs, ObsSnapshot):
+            raise TypeError(f'Expected ObsSnapshot, got {type(obs).__name__}')
+        snapshot = obs
 
         # Apply offset via OffsetFOV; metadata uses (dv, du)
-        try:
-            if 'offset' not in nav_metadata:
-                raise ValueError(f'{image_path}: "offset" field not found in metadata')
+        if 'offset' not in nav_metadata:
+            raise ValueError(f'{image_path}: "offset" field not found in metadata')
```

Ruff (0.14.10)

Avoid specifying long messages outside the exception class

(TRY003)